### PR TITLE
Change modal settings styles.

### LINKED ISSUE #4576: Settings screen not scaled on devices with narrow screen (<450px)
screenshot from 2017-10-26 13-56-05

### PR DESCRIPTION
**Pull request by Tainan404**
Closes #4576.
-  Change focus effect.
-  Leaves responsive.

peek 2017-12-19 09-35

### CODE REVIEW

**Comment by OZhurbenko** [comment]
Is it temporarily, or we changed our designs?
I think it looked a bit differently on the [picture](https://github.com/bigbluebutton/bigbluebutton/issues/4576#issuecomment-348366198)

**Comment by Tainan404** [comment]
@OZhurbenko, it is temporary.

File: bigbluebutton-html5/imports/ui/components/settings/styles.scss [hunk 1]
```diff
@@ -4,55 +4,83 @@
...
   display: flex;
   flex-flow: column;
-  flex-grow: 0;
-  border: none !important;
-  flex-shrink: 0;
-  border-right: 0.2rem solid #d4d9df;
+  border: none;
+  padding: 0;
+  margin: 0;
+  width: calc(100% / 3 * 1);
```
Comment (Gcampes) [inline bigbluebutton-html5/imports/ui/components/settings/styles.scss]: `calc(100% / 3);` (?)

**Comment by MaximKhlobystov** [comment]
Tested it locally, looks good. The code is pretty straightforward too.

**Pull request merged**